src/utility.ts

```diff
@@ -239,18 +227,14 @@ export class Utility {
         });
     }
 
-    public static async getModuleTwin(iotHubConnectionString: string, deviceId: string, moduleId: string): Promise<string> {
-        const url = `/twins/${encodeURIComponent(deviceId)}/modules/${moduleId}?api-version=${Constants.IoTHubApiVersion}`;
-        const config = Utility.generateIoTHubAxiosRequestConfig(iotHubConnectionString, url, "get");
-
-        return (await axios.request(config)).data;
+    public static async getModuleTwin(iotHubConnectionString: string, deviceId: string, moduleId: string): Promise<Twin> {
+        const registry: Registry = Registry.fromConnectionString(iotHubConnectionString);
+        return ((await registry.getModuleTwin(deviceId, moduleId)) as ResultWithHttpResponse<Twin>).responseBody;

```

Comment:
`getModuleTwin(deviceId: string, moduleId: string, done?: HttpResponseCallback<Twin>): Promise<ResultWithHttpResponse<Twin>> | void;`

It is `Promise<ResultWithHttpResponse<Twin>> | void`, seems I need to indicate it is `esultWithHttpResponse<Twin>`, otherwise there would be lint error. Since the return type could be `void`

Reply:
shall we handle the case void happens?

Reply:
If we don't pass `HttpResponseCallback<Twin>`, it will always return `Promise<ResultWithHttpResponse<Twin>>`